src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -660,8 +658,7 @@
   </Target>
 
   <Target Name="EnsureEmptyPackage"
-          DependsOnTargets="DetermineRuntimeDependencies"
-          BeforeTargets="GenerateNuSpec">
```

Comment:
Is there a growing resistance force against the use of BeforeTargets / AfterTargets?  ie, is there some intrinsic property of DependsOnTargets which makes that preferable, or is it just an easier mental model to consistently utilize the simpler inferred behavior of DependsOnTargets?